test/auth-react-server/index.js

```diff
@@ -292,16 +305,16 @@ app.post("/changeEmail", async (req, res) => {
 app.get("/unverifyEmail", verifySession(), async (req, res) => {
     let session = req.session;
     await EmailVerification.unverifyEmail(session.getRecipeUserId());
-    await session.fetchAndSetClaim(EmailVerification.EmailVerificationClaim);
+    await session.fetchAndSetClaim(EmailVerification.EmailVerificationClaim, {});
```

Comment:
why do we need to add `{}`?

Reply:
we don't need it, I just copied it from the version in st-auth-react to be consistent.